src/storhaug

```diff
@@ -324,15 +324,28 @@ setup_cluster()
 
     pcs cluster auth ${HA_SERVERS} -u hacluster -p ${HA_PASSWORD} --force
     pcs cluster setup --force --name ${HA_NAME} ${HA_SERVERS} || storlog "ERR" "Failed to setup cluster ${HA_NAME}"
+
+    # BZ 1284404, 1425110, allow time for SSL certs to propagate, until then
+    # pcsd will not accept connections.
+    sleep 12
     pcs cluster start --all || storlog "ERR" "Failed to start cluster ${HA_NAME}"
+    while [ $? -ne 0 ]; do
+        sleep 2
+        pcs cluster start --all || storlog "ERR" "Failed to start cluster ${HA_NAME}"
+    done
```

Comment:
But what happens if the command never succeeds? What if the cluster barfs mid-startup?